### ISSUE TITLE
software-store and software-updater icons differ only in color

### ISSUE THREAD

**Issue by matthewpaulthomas** [snwh/suru-icon-theme#91]
yaru-theme-* 18.10.4, Ubuntu Cosmic

The [software-store](https://github.com/snwh/suru-icon-theme/blob/master/src/fullcolor/apps/software-store.svg) and [software-updater](https://github.com/snwh/suru-icon-theme/blob/master/src/fullcolor/apps/software-updater.svg) icons appear to be identical except for their color scheme.

![icons for ubuntu software and software updater](https://user-images.githubusercontent.com/19801137/45940824-1409ee80-bfd3-11e8-9fb5-ac330871bc26.png)

This makes it impractical to guess which is which. And if you are color-blind, it may also be difficult to tell which is which even if you know the difference.

If the “A” is supposed to represent applications, then perhaps the software-updater icon should not consist of that motif, because (in Ubuntu at least) Software Updater provides updates for all software, not just apps.

[[Adapted from yaru#862](https://github.com/ubuntu/yaru/issues/862).]

**Comment by snwh** [comment]
Yep, it's poorly executed by me. It's an icon that was rushed out because it wasn't in the set.
